nucypher/blockchain/eth/actors.py

```diff
@@ -496,12 +503,12 @@ def perform_round_1(self, ritual_id: int, timestamp: int, *args, **kwargs):
         try:
             transcript = dkg.generate_transcript(
                 ritual_id=ritual_id,
-                checksum_address=self.checksum_address,
+                checksum_address=self.transacting_power.account,
                 shares=len(ritual.nodes),
-                threshold=ritual.threshold,
+                threshold=len(ritual.nodes) // 2 + 1,  # TODO: Make this configurable and reachable via contract
```

Comment:
yeah threshold isn't actually a parameter on a ritual at the mo (in the smart contract) 

i've commented on the PR